### PR TITLE
 Confirm maintainership for etcd website

### PR DESCRIPTION
**Pull request by jmhbnz**
Currently it is not fully clear who is an etcd website maintainer because:

- our github etcd maintainers-website team does not have any members populated, refer: https://github.com/orgs/etcd-io/teams/maintainers-website
- there is no `MAINTAINERS` file included at the root level of this repo.

This can create friction for pull requests because contributors don't know who they need to ping if they have questions or want help.

As a starting point I propose we include a `MAINTAINERS` file specifying all members of `etcd-io/maintainers-etcd`  as website maintainers.

Relates to https://github.com/etcd-io/website/issues/679

### CODE REVIEW

**Comment by jmhbnz** [comment]
Raised https://github.com/etcd-io/website/issues/692 to sort out why deploy previews are failing recently.

**Review by chalin — CHANGES_REQUESTED** [review]
Why not instead populate the existing team [maintainers-website](https://github.com/orgs/etcd-io/teams/maintainers-website) -- or better yet, use the name `website-maintainers`.  That would create a semantically meaningful and useful ID that we could use in CODEOWNERS files, or assign issues to, cc all maintainers, etc.

FYI, issue #692 has been fixed.

/cc @nate-double-u

**Comment by jmhbnz** [comment]
> Why not instead populate the existing team [maintainers-website](https://github.com/orgs/etcd-io/teams/maintainers-website) -- or better yet, use the name `website-maintainers`. That would create a semantically meaningful and useful ID that we could use in CODEOWNERS files, or assign issues to, cc all maintainers, etc.

Hey @chalin, thanks for fixing the netlify previews! 

Refer linked issue #679 where I am proposing that the github team be populated.  In relation to the actual issue of unclear maintainership I am not particularly worried which approach gets taken, file based or github team based, provided one is completed to remove the ambiguity.  I raised this pr to try and spur some conversation and hopefully get one or both implemented.

Note there has been a recent trend to rollout `MAINTAINERS` to all etcd-io subprojects starting recently with raft and bbolt.

**Comment by ahrtr** [comment]
Thanks @jmhbnz for raising the PR. Two comments:
1. Not sure why the workflow checks are failed. Could you resolve the failures?
2. I see the the following three contributors are also has admin or maintain role of this repo. But I am a little surprised that they are not even a member of etcd org.
    - @nate-double-u (document writer for etcd. @serathius mentioned this before) 
    - @chalin (already an admin of this repo, I did not know it before.) 
    - @jberkus (already has role Maintain of this repo, I did not know it before. But I see @jberkus has been contributing to etcd's doc, thx)

**Comment by jmhbnz** [comment]
> 1. Not sure why the workflow checks are failed. Could you resolve the failures?

Just needed to merge in the fix to the workflows which was completed under #692.

> 2. I see the the following three contributors are also has admin or maintain role of this repo. But I am a little surprised that they are not even a member of etcd org.

Please let me know if you would like to add any names to the list, keen to clear up any ambiguity :)

**Comment by ahrtr** [comment]
> Please let me know if you would like to add any names to the list, keen to clear up any ambiguity :)

Suggest to add all of them into the maintainer list. I am just curious why don't we follow the already established [membership guide](https://github.com/etcd-io/etcd/blob/main/Documentation/contributor-guide/community-membership.md).

File: MAINTAINERS
Comment (chalin) [inline MAINTAINERS:19]:
```suggestion
Patrice Chalin <chalin@cncf.io> (@chalin) *
```

**Review by chalin — APPROVED** [review]
Let's use my CNCF email address if possible. Thx

**Comment by jmhbnz** [comment]
> @ahrtr agree, it's not clear. @nate-double-u and @chalin are helping from CNCF side and they led the website redesign work after etcd project graduation in the past. Now that we are going to have a maintainers file specifically for this repo, we should add them and Josh (after talking to them) in this PR or in a separate one. Thanks! @jmhbnz thanks for working on this PR!

@jberkus, @nate-double-u, @chalin -  Before we proceed further with this pr can you please respond below if you are in agreement to be formally added as a maintainer for `etcd-io/website`, acknowledging the [requirements and responsibilities](https://github.com/etcd-io/etcd/blob/main/Documentation/contributor-guide/community-membership.md#requirements-2) this entails?

**Review by spzala — COMMENTED** [review]
Thanks @jmhbnz A small things on convention, please add new members in an alphabetical order. Thanks!

**Review by ahrtr — APPROVED** [review]
LGTM

Thanks @jmhbnz 

Once this PR gets merged, we can add all maintainers into github team website-maintainers?

**Review by spzala — APPROVED** [review]
@jmhbnz thanks for addressing my comment quickly!

**Comment by jberkus** [comment]
Yes.  That said, I'm good at editing content but fairly poor at troubleshooting netlify stuff, so I hope that we have access to @chalin for some time to come.

File: MAINTAINERS
Comment (nate-double-u) [inline MAINTAINERS:17]: If permitted, I prefer:
```suggestion
Nate W. <natew@cncf.io> (@nate-double-u) *
```

Comment (spzala) [reply MAINTAINERS:17]: @nate-double-u of course, that's fine! I don't see any issue with that. cc @jmhbnz  Thanks!

Comment (jmhbnz) [reply MAINTAINERS]: Sounds good team, updated 👍🏻

Comment (spzala) [reply MAINTAINERS]: Thanks @jmhbnz

**Review by nate-double-u — APPROVED** [review]

File: MAINTAINERS
Comment (nate-double-u) [reply MAINTAINERS]: Thanks!

**Comment by chalin** [comment]
> @chalin - Before we proceed further with this pr can you please respond below if you are in agreement to be formally added as a maintainer for `etcd-io/website`, acknowledging the [requirements and responsibilities](https://github.com/etcd-io/etcd/blob/main/Documentation/contributor-guide/community-membership.md#requirements-2) this entails?

@jmhbnz - yes, I agree to be added as a **maintainer for the website** portion of the etcd project, in the mainly role of **web dev and technical writer**, including support for **Google Analytics (4) and Netlify**. My tech writer role would be mainly for Information Architecture (site org) recommendations, and some limited contributions content (again, mostly reorg, copyediting). Like Nate, I have not registered to the various lists. Note that I'll be more involved in etcd.io website work as my responsibilities of other projects diminish -- it's looking like that will happen in Q3. Thanks!

**Review by ahrtr — APPROVED** [review]

**Comment by ahrtr** [comment]
cc @mitake @ptabor @serathius @wenjiaswe

File: MAINTAINERS
Comment (wenjiaswe) [inline MAINTAINERS:21]: Could you please add me there as well?
Wenjia Zhang <wenjiazhang@google.com> <wenjia.swe@gmail.com> (@wenjiaswe) *

Thanks you very much!

Comment (jmhbnz) [reply MAINTAINERS]: Done 👍🏻

**Comment by jmhbnz** [comment]
Note following https://github.com/etcd-io/etcd/pull/16234 I have also added myself as reviewer here.

**Comment by ahrtr** [comment]
ping @wenjiaswe and @serathius

File: MAINTAINERS
Comment (wenjiaswe) [inline MAINTAINERS:21]: nits: add "<>" around the emails?

Otherwise lgtm.

Thanks!

Comment (nate-double-u) [reply MAINTAINERS:21]:
```suggestion
Wenjia Zhang <wenjiazhang@google.com> <wenjia.swe@gmail.com> (@wenjiaswe) *
```

Comment (jmhbnz) [reply MAINTAINERS]: Good spotting, done, thanks!

**Comment by ahrtr** [comment]
With the approval of 5 maintainers and 1 reviewer, merging this PR...

Afterwards, I will add all maintainers into maintainers-website.

**Comment by ahrtr** [comment]
@jberkus @nate-double-u @chalin I have already invited you to join the etcd-io org.

**Pull request merged**
